backend/modules/cohorts/repository.py

```
@@ -0,0 +1,547 @@
+            func.count(CohortStudent.id).label('student_count')
+        ).filter(
+            CohortStudent.status == "approved"
+        ).group_by(CohortStudent.cohort_id).subquery()
+        
+        simulation_count_subquery = self.db.query(
+            CohortSimulation.cohort_id,
+            func.count(CohortSimulation.id).label('simulation_count')
+        ).join(
+            Scenario, CohortSimulation.simulation_id == Scenario.id
+        ).filter(
+            Scenario.is_draft == False,
```

Ruff (0.14.8)

Avoid equality comparisons to `False`; use `not Scenario.is_draft:` for false checks

Replace with `not Scenario.is_draft`

(E712)